test/Microsoft.Framework.Runtime.FunctionalTests/Utilities/PathHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace Microsoft.Framework.Runtime.FunctionalTests.Utilities
+{
+    public static class PathHelper
+    {
+        public static string NormalizeSeparator(string path)
+        {
+            return path.Replace('\\', Path.DirectorySeparatorChar)
```

Comment:
No, `AltDirectorySeparatorChar` is not necessary `\\` on *nix. 

The value of this field is a backslash ('\') on UNIX, and a slash ('/') on Windows and Macintosh operating systems.